pkg/util/promqlext/promqlext.go

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package promqlext
+
+import (
+	"github.com/prometheus/prometheus/promql"
+	"github.com/prometheus/prometheus/promql/parser"
+)
+
+// ExtendPromQL enriches PromQL with Mimir extensions.
+func ExtendPromQL() {
+	// Keep an alias for users using holt_winters, even though dropped in Prometheus v3.
+	promql.FunctionCalls["holt_winters"] = promql.FunctionCalls["double_exponential_smoothing"]
+	parser.Functions["holt_winters"] = parser.Functions["double_exponential_smoothing"]
+	parser.Functions["holt_winters"].Experimental = false
```

Comment:
We're not removing `double_exponential_smoothing` from Mimir, so it's a new function for Mimir users, so it should be mentioned in the changelog.

Reply:
Added it to changelog, thanks.